bin/ch/WScriptJsrt.cpp

```diff
@@ -808,8 +808,10 @@ WScriptJsrt::CallbackMessage::~CallbackMessage()
     {
         WScriptJsrt::PrintException("", JsErrorScriptException);
     }
+CLANG_WNO_BEGIN("-Wunused-variable")
     JsErrorCode errorCode = ChakraRTInterface::JsRelease(m_function, nullptr);
```

Comment:
Can we just change this to Assert(ChakraRTInterface::JsRelease(m_function, nullptr) == JsNoError) ?